maths/series/arithmetico_geometric_sequence.py

```diff
@@ -0,0 +1,149 @@
+    Sum Of Your A.G.S. : 35.00
+    Infinite Series Sum : 40.00
+    Value Of Which Term You Want : 2
+    Value Of 2-th Term : 10.00
+    """
+
+    def full_series(self):
+        series = []
+        for i in range(self.n):
+            if i == 0:
+                series.append(str(self.a) + " x " + str(self.b))
+            else:
+                ap_part = str(self.a) + " + " + str(i * self.d)
+                gp_part = str(self.b) + "x" + str(self.r) + "^" + str(i)
+                series.append("(" + ap_part + ")" + " x " + gp_part)
+        return series
+
+    def last_term_value(self):
+        return (self.a + (self.n - 1) * self.d) * (self.b * pow(self.r, self.n - 1))
+
+    def sum(self):
+        return (
+            (self.a * self.b)
+            - ((self.a + (self.n * self.d)) * (self.b * pow(self.r, self.n)))
+        ) / (1 - self.r) + (self.d * self.b * self.r * (1 - pow(self.r, self.n))) / pow(
+            (1 - self.r), 2
+        )
+
+    def inf_sum(self):
+        return (self.a * self.b) / (1 - self.r) + (self.d * self.b * self.r) / pow(
+            (1 - self.r), 2
+        )
+
+    def nth_term_value(self, k):
+        return (self.a + (k - 1) * self.d) * (self.b * pow(self.r, k - 1))
+
+
+def main():
+    run = True
+    while run:
+        try:
+            a = float(input("\nEnter Initial Value For A.P. : "))
+            run = False
+        except ValueError:
+            print("Please Give A Number For Corresponding Input!")
+    run = True
+    while run:
+        try:
+            d = float(input("Enter Difference For A.P. : "))
+            run = False
+        except ValueError:
+            print("Please Give A Number For Corresponding Input!")
+    run = True
+    while run:
+        try:
+            b = float(input("Enter Initial Value For G.P. : "))
+            run = False
+        except ValueError:
+            print("Please Give A Number For Corresponding Input!")
+
+    run = True
+    while run:
+        try:
+            r = float(input("Enter Common Ratio For G.P. : "))
+            run = False
+        except ValueError:
+            print("Please Give A Number For Corresponding Input!")
+    not_get_n = True
+    while not_get_n:
+        try:
+            n = float(input("Enter Number Of Terms : "))
+            if n == int(n):
+                n = int(n)
+                not_get_n = False
+            else:
+                print("Please Give An Integer For Corresponding Input!")
+        except ValueError:
+            print("Please Give An Integer As The Number Of Terms!")
+
+    ags = AGS(a, d, b, r, n)
+
+    print("\nFull Series : \n{}".format(ags.full_series()))
+    print("\nValue Of Last Term : {:.2f}".format(ags.last_term_value()))
+    print("Sum Of Your A.G.S. : {:.2f}".format(ags.sum()))
+    print("Infinite Series Sum : {:.2f}".format(ags.inf_sum()))
+
+    not_get_k = True
+    while not_get_k:
+        try:
+            k = float(input("\nValue Of Which Term You Want : "))
```

Comment:
If you want an integer, why convert it to a float??  Just convert directly to int.